src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,8 @@
+package wolox.training.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import wolox.training.models.Book;
+
+public interface BookRepository  extends JpaRepository<Book, Long> {
```

Comment:
Add the @Repository annotation on the interface.

```suggestion
@Repository
public interface BookRepository  extends JpaRepository<Book, Long> {
```

Reply:
I added @repository in this method